frontend/public/components/deploy-image.tsx

```diff
@@ -397,6 +403,13 @@ export class DeployImage extends React.Component<DeployImageProps, DeployImageSt
                   <div className="help-block">Identifies the resources created for this image.</div>
                 </div>
               </div>
+              <div className="form-group co-deploy-image__labels">
+                <label htmlFor="tags-input" className="control-label">Labels</label>
+                <SelectorInput labelClassName="co-text-deploymentconfig" onChange={this.onLabelsChange} tags={labels} />
+                <div className="help-block" id="labels-help">
+                  {`If no labels are specified, app=${name || '<name>'} will be added.`}
```

Comment:
Missed this in the other PR, but...

```suggestion
                  If no labels are specified, app={name || '<name>'} will be added.
```

Reply:
Thanks.  Fixed.